compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -2122,19 +2122,34 @@ object SymDenotations {
         case pcls :: pobjs1 =>
           if (pcls.isCompleting) recur(pobjs1, acc)
           else {
-            // A package object inherits members from `Any` and `Object` which
-            // should not be accessible from the package prefix.
             val pmembers = pcls.computeNPMembersNamed(name).filterWithPredicate { d =>
+              // Drop members of `Any` and `Object`, as well as top-level definitions
+              // in the empty package that are not defined in the current run.
```

Comment:
I don't think the code below is related to dropping definitions in the empty package ?

Reply:
Oops, that was left from the previous abandoned PR.